_topic_maps/_topic_map.yml

```diff
@@ -927,37 +927,47 @@ Topics:
 - Name: Compliance Operator
   Dir: compliance_operator
   Topics:
+  - Name: Compliance Operator overview
+    File: co-overview
   - Name: Compliance Operator release notes
     File: compliance-operator-release-notes
-  - Name: Supported compliance profiles
-    File: compliance-operator-supported-profiles
-  - Name: Installing the Compliance Operator
-    File: compliance-operator-installation
-  - Name: Updating the Compliance Operator
-    File: compliance-operator-updating
-  - Name: Compliance Operator scans
-    File: compliance-scans
-  - Name: Understanding the Compliance Operator
-    File: compliance-operator-understanding
-  - Name: Managing the Compliance Operator
-    File: compliance-operator-manage
-  - Name: Tailoring the Compliance Operator
-    File: compliance-operator-tailor
-  - Name: Retrieving Compliance Operator raw results
-    File: compliance-operator-raw-results
-  - Name: Managing Compliance Operator remediation
-    File: compliance-operator-remediation
-  - Name: Performing advanced Compliance Operator tasks
-    File: compliance-operator-advanced
-  - Name: Troubleshooting the Compliance Operator
-    File: compliance-operator-troubleshooting
-  - Name: Uninstalling the Compliance Operator
-    File: compliance-operator-uninstallation
-  - Name: Using the oc-compliance plugin
-    File: oc-compliance-plug-in-using
-  - Name: Understanding the Custom Resource Definitions
-    File: compliance-operator-crd
-
+  - Name: Compliance Operator concepts
+    Dir: co-concepts
+    Topics:
+    - Name: Understanding the Compliance Operator
+      File: compliance-operator-understanding
+    - Name: Understanding the Custom Resource Definitions
+      File: compliance-operator-crd
+    - Name: Managing the Compliance Operator
+      File: compliance-operator-manage
+  - Name: Compliance Operator management
+    Dir: co-management
+    Topics:
+    - Name: Installing the Compliance Operator
+      File: compliance-operator-installation
+    - Name: Updating the Compliance Operator
+      File: compliance-operator-updating
+    - Name: Troubleshooting the Compliance Operator
+      File: compliance-operator-troubleshooting
+    - Name: Uninstalling the Compliance Operator
+      File: compliance-operator-uninstallation
+  - Name: Compliance Operator scan management
```

Comment:
Are we going to have separate sections for other CO objects, like `Remediations` or `TailoredProfiles`? If so - `Profiles` and `TailoredProfiles` seem like a logical grouping. `ScanSettings` and `ScanSettingBindings` seem like that would be appropriate in this section. Then another section for working with the results?

I'm not sure if that's where you're going, just curious.

Reply:
I think that is a logical next step! I would love to chat more about this at any time!